pastis/matrix_generation/matrix_from_efields.py

```diff
@@ -134,6 +141,44 @@ def setup_single_mode_function(self):
         self.calculate_one_mode = functools.partial(_luvoir_matrix_single_mode, self.number_all_modes, self.wfe_aber,
                                                     self.luvoir, self.resDir, self.save_efields, self.saveopds)
 
+class MatrixEfieldRST(PastisMatrixEfields):
+    """
+    Class to calculate RST Electrical field of RST CGI.
+    """
+    instrument = 'RST'
+
+    def __init__(self, max_local_zernike=3, initial_path='', saveefields=True, saveopds=True):
+        super().__init__(initial_path=initial_path, saveefields=saveefields, saveopds=saveopds)
+        self.max_local_zernike = max_local_zernike
```

Comment:
You don't need this input parameter or attribute (`max_local_zernike`) in the RST case, so delete all three of them.

Reply:
@davidbourgeois you still need the `def __init__()` ! 😱

Reply:
Yes, I understand but, worked. I correct it

Reply:
Yeah I just realized that even without it would just use the inherited `__init__()` method, which is correct, but I think it's worth to keep things consistent between the individual child classes. Refactors should happen separately from new features, to avoid bugs.

Reply:
I can't commit it...

Reply:
Why not, what's happening?

Reply:
A stash blocked tree. I removed it, works now.